### PR TITLE
Feature/display user list for admins

### PR DESCRIPTION
**Pull request by nilssonemil**

### CODE REVIEW

File: src/app/admin/components/user-permission-table/user-permission-table.component.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<div class="table-container">
+  <table mat-table [dataSource]="dataSource">
+    <ng-container matColumnDef="alias">
+      <th mat-header-cell *matHeaderCellDef>Alias</th>
+      <td mat-cell *matCellDef="let user">{{ user.alias }}</td>
+    </ng-container>
+    <ng-container matColumnDef="id">
```
Comment (hansenemily) [inline src/app/admin/components/user-permission-table/user-permission-table.component.html:7]: What use is ID to the user? Will that tell the user anything? Or is it just here for now and will be removed later on when more information is available?

File: src/app/app.module.ts [hunk 2]
```diff
@@ -19,7 +19,6 @@ import {
   MatIconModule,
   MatSelectModule,
   MatExpansionModule,
-  MatTooltip,
```
Comment (hansenemily) [inline src/app/app.module.ts:22]: Why is this removed?

File: src/app/resolvers/user-list.resolver.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core'
```
Comment (hansenemily) [inline src/app/resolvers/user-list.resolver.ts:1]: Why do we need a user-list.resolver? Why not call user.service straight from user-permission-management? There's a lot of layers atm and I'm just want to know why :)

**Review by hansenemily — COMMENTED** [review]
Will you move admin-panel to admin as well? Or what should be in admin/not?

File: src/app/admin/components/user-permission-table/user-permission-table.component.html (hunk 1 above)
Comment (nilssonemil) [reply src/app/admin/components/user-permission-table/user-permission-table.component.html:7]: Not sure if the ID is going to be useful. For now it helps to give an overview over all users and their data I suppose. If this view becomes bloated maybe it can be removed.

File: src/app/app.module.ts (hunk 2 above)
Comment (nilssonemil) [reply src/app/app.module.ts:22]: Should probably be done in its own commit (I had missed this). But the only that should be needed to be included from material ui in app.module.ts are *Module* files. See e.g. MatExpansionModule, MatSelectModule etc. I assume we have a MatTooltipModule somewhere that allows us to use MatTooltip in our components.

File: src/app/resolvers/user-list.resolver.ts (hunk 3 above)
Comment (nilssonemil) [reply src/app/resolvers/user-list.resolver.ts:1]: I'm not sure if it is necessarily a good practice, but right now I'm in favour of it :P

Specifying resolvers for routes allows us to resolve our data before we start loading the component. This means we do not need to have any logic for when no data is available/being fetched etc. I suppose this would have greater benefits when we are not dealing with lists of data, but rather an object's detailed view (as those are trickier to display without data).

**Comment by nilssonemil** [comment]
> Will you move admin-panel to admin as well? Or what should be in admin/not?

My intention is to move anything related to the administration panel to the admin module. I just took a few things I touched first.

File: src/app/resolvers/user-list.resolver.ts (hunk 3 above)
Comment (hansenemily) [reply src/app/resolvers/user-list.resolver.ts:1]: It's just four different parts to do one thing, I guess it will become clearer when we have some more data.

**Review by hansenemily — APPROVED** [review]

File: src/app/admin/components/user-permission-table/user-permission-table.component.html (hunk 1 above)
Comment (hansenemily) [reply src/app/admin/components/user-permission-table/user-permission-table.component.html:7]: I guess it might become confusing for the user later on, if you remove some users and therefor lack some of the numbers. But yeah, it can wait until we have more information about each user

**Comment by nilssonemil** [comment]
This PR seems kinda dead @hansenemily
